pyproject.toml

```
@@ -42,6 +42,7 @@ Website = "https://pipecat.ai"
 [project.optional-dependencies]
 anthropic = [ "anthropic~=0.49.0" ]
 assemblyai = [ "websockets~=13.1" ]
+asyncai = [ "websockets~=13.1" ]
```

We've recently bumped to the websocket version support and now use the asyncio implementation.

```suggestion
asyncai = [ "websockets>=13.1,<15.0" ]
```